tools/get_config.py

```diff
@@ -54,9 +54,7 @@
     target = extract_mcus(parser, options)[0]
 
     # Toolchain
-    if options.tool is None:
-        args_error(parser, "argument -t/--toolchain is required")
-    toolchain = options.tool[0]
+    toolchain = options.tool[0] if options.tool is not None else None
```

Comment:
Any reason this couldn't be shrunk to `options.tool[0] if options.tool else None`?

Reply:
No. Originally we had to worry about empty strings and what not.